boards/nucleo-wl55jc/Makefile.features

```diff
@@ -11,6 +11,3 @@ FEATURES_PROVIDED += periph_uart
 
 # Put other features for this board (in alphabetical order)
 FEATURES_PROVIDED += riotboot
-
-# load the common Makefile.features for Nucleo boards
-# include $(RIOTBOARD)/common/nucleo64/Makefile.features
```

Comment:
Why was this removed?

Reply:
The CI was giving error ( `arduino` ) due to mismatch b/w `Kconfig` and `Makefile.features`.

Reply:
The add the features? This seems like a workaround instead of a proper fix, there is probably a Kconfig file inclusion missing.

Reply:
Okay. I shall `arduino.h` headers now.